privval/file.go

```diff
@@ -182,6 +182,12 @@ func GenFilePV(keyFilePath, stateFilePath, keyType string) (*FilePV, error) {
 	}
 }
 
+// RecoverFilePV recovers a validator from the given private key
+// and sets the filePaths, but does not call Save().
+func RecoverFilePV(privKey crypto.PrivKey, keyFilePath, stateFilePath string) *FilePV {
```

Comment:
why can't we use existing NewFilePV?

Reply:
@melekes My brain messed up. Idk what happened there lol. Going to close this